packages/legend-application-studio/src/stores/editor-state/element-editor-state/connection/ConnectionEditorState.ts

```diff
@@ -102,6 +111,9 @@ export class RelationalDatabaseConnectionValueState extends ConnectionValueState
   override connection: RelationalDatabaseConnection;
   selectedTab = RELATIONAL_DATABASE_TAB_TYPE.GENERAL;
   databaseBuilderState: DatabaseBuilderState;
+  selectedPostProcessor: PostProcessor | undefined;
```

Comment:
As mentioned in `PostProcessorEditorState`, let's make this `selectedPostProcessorState?: PostProcessorEditorState | undefined` instead